client/coral-admin/src/containers/ModerationQueue/UserDetail.js

```diff
@@ -0,0 +1,60 @@
+import React, {PropTypes} from 'react';
+import {Button, Drawer} from 'coral-ui';
+import styles from './UserDetail.css';
+import {compose} from 'react-apollo';
+import {getUserDetail} from 'coral-admin/src/graphql/queries';
+
+class UserDetail extends React.Component {
+  static propTypes = {
+    id: PropTypes.string.isRequired,
+    hideUserDetail: PropTypes.func.isRequired
+  }
+
+  copyPermalink () {
+    this.profile.select();
+    try {
+      document.execCommand('copy');
+    } catch (e) {
+
+      /* nothing */
+    }
+  }
+
+  render () {
+    const {data, hideUserDetail} = this.props;
+
+    if (!('user' in data)) {
+      return null;
+    }
+
+    const {user} = data;
+    const localProfile = user.profiles.find(p => p.provider === 'local');
+    const facebookProfile = user.profiles.find(p => p.provider === 'facebook');
```

Comment:
This should not reference plugins content. We should instead utilize slots to inject Facebook details into the user detail pane.